ansible/playbooks/roles/applications/templates/auth-service/00-auth-service.yml.j2

```diff
@@ -209,6 +209,10 @@ spec:
               name: "{{ auth_service_name }}-db"
         - name: X509_CA_BUNDLE
           value: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+{% if data.service.proxy_address_forwarding is defined and data.service.proxy_address_forwarding %}
```

Comment:
`if data.service.proxy_address_forwarding is defined` - should it be checked if there is alvays a default value?

Reply:
From Epicli perspective this default should always be there but when we move to the collections as people might chose not to run through CLI.  So I like to cover the bases.

Reply:
default(false) filter could be used, but current version is of for me